### PR TITLE
ci(performance): create initial gha workflow [INFENG-224]

### PR DESCRIPTION
**Pull request by loksonarius**
## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: .github/workflows/performance-tests.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        continue-on-error: true
+        run: make -C performance run
+
+      - name: Persist cluster if failure noticed
+        if: failure()
+        run: make -C performance persist
+
+      - name: Depersist cluster if tests passed
+        if: failure()
+        run: make -C performance persist
```
Comment (dannysauer) [inline .github/workflows/performance-tests.yml:60]: This feels incorrect. :)

File: .github/workflows/performance-tests.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+on:
+  # to iterate on during PR
+  pull_request: {}
+  schedule:
+    - cron: '0 2 * * 0-4'
+
+env:
+  AWS_ACCESS_KEY_ID: ${{ secrets.PERFORMANCE_AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.PERFORMANCE_AWS_SECRET_ACCESS_KEY }}
+  AWS_DEFAULT_REGION: us-west-2
```
Comment (dannysauer) [inline .github/workflows/performance-tests.yml:12]: How do you feel about a less-used region like us-east-2? Or are you using a different account than the main sandbox?

Comment (loksonarius) [reply .github/workflows/performance-tests.yml:12]: Sure, I'll try deploying to us-east-2

**Comment by loksonarius** [comment]
So all the required wiring for CI is good any ready, but to be more useful outside, we should really get the generated reporting fixed up so the data shows up both human-readable in Slack and dev-friendly in GitHub.
Both of these tasks are looking a bit more involved than what I had hoped to get done as part of this work, so I'll be doing that in a future PR.

**Review by julian-determined-ai — APPROVED** [review]

**Review by julian-determined-ai — APPROVED** [review]

**Pull request merged**